simpl/rest.py

```diff
@@ -23,6 +23,77 @@
 STANDARD_QUERY_PARAMS = ('offset', 'limit', 'sort', 'q', 'facets')
 
 
+def One(schema=str):
+    """Expect the input sequence to contain a single value.
+
+    :keyword schema:
+        Custom schema to apply to the input value. Defaults to just string,
+        since this is designed for query params.
+    """
+
+    def f(v):
+        """"""
+        [value] = v
+        return schema(value)
+    return f
+
+
+def Many(schema=str):
```

Comment:
The purpose of `Many` is to specify a sequence of values for bottle query params. After looking at this again, it could be replaced with `[str]` or `[sometype]`.